app/src/main/java/com/mathewsachin/fategrandautomata/ui/skill_maker/SkillMakerKukulcan.kt

```diff
@@ -0,0 +1,65 @@
+package com.mathewsachin.fategrandautomata.ui.skill_maker
+
+import androidx.compose.foundation.layout.*
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.colorResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.mathewsachin.fategrandautomata.R
+import com.mathewsachin.fategrandautomata.scripts.models.ServantTarget
+
+@Composable
+fun SkillMakerKukulcan(
+    onOption1: () -> Unit,
+    onOption2: () -> Unit,
+    goToTarget: Boolean,
+    onTarget: (firstTarget: ServantTarget) -> Unit
+) {
+    Column(
+        modifier = Modifier
+            .fillMaxHeight()
+            .padding(16.dp)
+    ) {
+        Text(
+            stringResource(R.string.skill_maker_kukulcan),
```

Comment:
For some reason, this text is black in dark mode, so you can't read it.

Space Ishtar and Emiya don't have this problem but I can't find out why.

I manually specified `color = onSurface` in `FGATitle` and replaced the `Text()` calls with `FGATitle()`.